src/Ch9/Ch9.Shared/Client/ClientConstants.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Ch9.Client
+{
+	public static class ClientConstants
+	{
+		public const string BaseApiUrl = "https://channel9-functionapp.azurewebsites.net/api";
```

Comment:
Any chance we can get a name that is simpler? Like ch9?

Reply:
We can do that, but that would need to re-create the app in azure, as we cannot rename an existing app.